packages/jszip-cli/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
Can we use `glob` without activating `esModuleInterop`?

Reply:
The explicit imports inside the `src`-folder can work without `esModuleInterop`, but then it's the implicit imports within dependencies that fail.

<details>
 <summary>Build Log</summary>

```sh
$ tsc
../../node_modules/path-scurry/dist/cjs/index.d.ts:4:8 - error TS1259: Module '"/Users/jan/Desktop/node-packages/node_modules/path-scurry/node_modules/lru-cache/index"' can only be default-imported using the 'esModuleInterop' flag

4 import LRUCache from 'lru-cache';
         ~~~~~~~~

  ../../node_modules/path-scurry/node_modules/lru-cache/index.d.ts:869:1
    869 export = LRUCache
        ~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

../../node_modules/path-scurry/dist/cjs/index.d.ts:7:8 - error TS1259: Module '"/Users/jan/Desktop/node-packages/packages/jszip-cli/node_modules/minipass/index"' can only be default-imported using the 'esModuleInterop' flag

7 import Minipass from 'minipass';
         ~~~~~~~~

  node_modules/minipass/index.d.ts:150:1
    150 export = Minipass
        ~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/glob/dist/mjs/glob.d.ts:3:8 - error TS1259: Module '"/Users/jan/Desktop/node-packages/packages/jszip-cli/node_modules/minipass/index"' can only be default-imported using the 'esModuleInterop' flag

3 import Minipass from 'minipass';
         ~~~~~~~~

  node_modules/minipass/index.d.ts:150:1
    150 export = Minipass
        ~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/glob/dist/mjs/index.d.ts:1:8 - error TS1259: Module '"/Users/jan/Desktop/node-packages/packages/jszip-cli/node_modules/minipass/index"' can only be default-imported using the 'esModuleInterop' flag

1 import Minipass from 'minipass';
         ~~~~~~~~

  node_modules/minipass/index.d.ts:150:1
    150 export = Minipass
        ~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/glob/dist/mjs/walker.d.ts:8:8 - error TS1259: Module '"/Users/jan/Desktop/node-packages/packages/jszip-cli/node_modules/minipass/index"' can only be default-imported using the 'esModuleInterop' flag

8 import Minipass from 'minipass';
         ~~~~~~~~

  node_modules/minipass/index.d.ts:150:1
    150 export = Minipass
        ~~~~~~~~~~~~~~~~~
    This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.


Found 5 errors in 4 files.
```

</details>

The only alternative I can think of is using a bundler rather than using `tsc` directly.

Reply:
Ah well.